app/views/applications/applications.scala.html

```diff
@@ -64,24 +79,21 @@ <h1>Applications</h1>
 
                     <div class="grid-layout__column--1-2">
                         <label class="form-label bold" for="filter">Filter by API Subscription
-                            <select class="form-control input-select input-select--large"
-                            data-datatable-filter="applications-table"
-                            id="filter"
-                            name="filter"
-                            value="">
-
-                                <option @if(filter == None || filter == Some("")){selected} value data-api-name>All applications</option>
-                                <option @if(filter == Some("ANYSUB")){selected} value="ANYSUB" data-api-name="ANYSUB">One or more subscriptions</option>
-                                <option @if(filter == Some("NOYSUB")){selected} value="NOSUB" data-api-name="NOSUB">No subscriptions</option>
+                            <select class="form-control input-select input-select--large" id="filter" name="apiSubscription" onchange="this.form.submit()">
```

Comment:
I think this onChange event (and others on the page) gives us an accessibility/keyboard use issue.